.github/workflows/rerun-workflows.yml

```diff
@@ -14,10 +14,7 @@ jobs:
     steps:
       - name: Generate job summary
         run: |
-          RUN_ID=${{ github.event.workflow_run.id }}
-          WORKFLOW_NAME=${{ github.event.workflow_run.name }}
-
-          echo "# $WORKFLOW_NAME workflow failed! :x:" >> $GITHUB_STEP_SUMMARY
+          echo "# ${{ github.event.workflow_run.name }} workflow failed! :x:" >> $GITHUB_STEP_SUMMARY
```

Comment:
1. ended up never using `RUN_ID`
2. since we're not going to reuse `WORKFLOW_NAME` and `github.even.workflow_run.name` it seems ok to use it directly inside this step

Reply:
Got it now, so it was failing when trying to declare `WORKFLOW_NAME`

```shell
WORKFLOW_NAME=E2E Tests
```